static/sass/styles.scss

```
@@ -642,3 +641,56 @@ $p-small-lh-diff: map-get($line-heights, default-text) - map-get($line-heights,
     }
   }
 }
+
+.p-icon--chevron-right {
+  @extend .p-icon--chevron-up;
+
+  transform: rotate(-90deg);
+}
+
+.p-engineering-fast-track {
+  margin-bottom: $spv--large;
+
+  &__container {
+    background-color: $color-light;
```

suggestion in the cod,e the fast track roles need to be white,  otherwise they blend in with the background:

```suggestion
    background-color: $color-x-light;
```